contrib/exporters/secadvisor/main.go

```diff
@@ -27,5 +27,6 @@ func main() {
 }
 
 func init() {
+	core.EncoderHandlers.Register("secadvisor_json", mod.NewEncodeSecadvsiorJSON, false)
```

Comment:
just call this encoder "sa"

Reply:
register it as the *default* encoder (2nd argument should be true)

Reply:
you also want to register this to exporters/uber so that it is available in the single unified binary

Reply:
Done, thanks.

I agree here to use `"sa"` for consistency, but generally I would not use this unclear abbreviation and would opt instead for `secadvisor` and similar. Consider for future.

Reply:
I am fine with using "secadvisor" consistently throughout

Reply:
done in 25308601b7fd97ec265b37730f23a32b4b43b828.